registry/storage/driver/walk.go

```diff
@@ -32,7 +34,14 @@ func WalkFallback(ctx context.Context, driver StorageDriver, from string, f Walk
 		// performance bottleneck.
 		fileInfo, err := driver.Stat(ctx, child)
 		if err != nil {
-			return err
+			switch err.(type) {
+			case PathNotFoundError:
+				// repository was removed in between listing and enumeration. Ignore it.
+				logrus.WithField("path", child).Infof("ignoring deleted path")
```

Comment:
Should this be info or debug?

Reply:
Good question. I was thinking since it should be a rare race, rather than a matter of course a user would be more interested to know when this happens, but _shrug_. @dmcgowan thoughts?

Reply:
I think Info is correct here